amy/templates/emails/email_template_create.html

```diff
@@ -6,4 +6,13 @@
 
 {% crispy form %}
 
+<div class="row">
+  <div class="col-12 col-lg-2">
+    <strong>Template variables</strong>
+  </div>
+  <div class="col-12 col-lg-10">
+    Visible after saving the template.
```

Comment:
Hmm, it would be nice to show the variables as soon as a signal is selected, but I know that would increase the complexity a lot (is it even possible for this case?)
I think we can manage with this and make it part of the workflow to save before entering the full body text, since we shouldn't be creating tons of templates.

Reply:
I think it's not impossible, but since I have a ton of other features to implement I didn't focus on this case.

One option I can see is to load the variables and their definitions to a JSON object, then switch display in this depending on what user is choosing in the dropdown.